.buildconfig.yml

```diff
@@ -1,4 +1,4 @@
-libraryVersion: 20.0.0-SNAPSHOT
```

Comment:
Mh, this looks weird. Are we switching back to `20.0.0-SNAPSHOT` after this PR?

Reply:
Sounds good to me. Let's merge this, take it back to 20, and then discuss this in the next Glean SDK meeting